### DISCUSSION TITLE
Deepfake技術詐騙💰︎

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#141] [General]
**新聞內容:**
**Deepfake詐騙風暴**來襲，台灣已有Deepfake假冒銀行詐騙案例，根據警政署統計，2021年1至10月詐欺案累計高達20,410件，較上年同期增加10.81%，上半年投資詐欺高達2,283件，較去年同期增加近一倍94.61%。
台灣雖尚未傳出以Deepfake AI技術偽冒**生物識別**（人臉、聲紋）進行身份驗證詐欺案件，然而，另一程度的金融版 Deep Fake 案件自年初以來頻傳，歹徒偽裝銀行行員、假冒證交所/財經專家進行投資詐欺、偽冒銀行服務簡訊/銀行網頁、來電顯示真實銀行電話騙取信任等，讓受害者難辨真偽而不慎落入陷阱。
目前詐騙集團已有竄改來電顯示號碼的技術，冒用證交所電話，來電以分享投資資訊為由，邀請民眾加入投資群組。使用財經專家頭像、知名媒體名義等，騙取民眾加入投資群組，四天內詐騙61人，詐騙金額上千萬元。
**新聞心得:**
我記得我第一次聽說Deepfake是在玩命關頭拍攝的時候使用保羅沃克的樣貌移轉到主角弟弟的身上拍攝，但沒想到近期關於這項技術的負面新聞報多，被沒良心的人使用在犯罪及詐騙上，而且騙到手的金額還不是小數目。
**新聞來源:**
https://udn.com/news/story/7320/5949915

**Comment by Timmy110403540** [comment]
不當利用這項技術有可能會造成很大的影響，如假冒一些官員的容貌來做壞事

**Comment by pocper1** [comment]
deepfake 在開發 勢必之後也要開發出一套軟體可以辨別是不是有使用deepfake技術

**Comment by briankuo123** [comment]
我相信Deepfake原本的目的應該是想要幫助社會朝向更好得方向邁進，只能說那些動歪腦筋的人真的是太多了。

**Reply by cobb110202008** [reply]
認同，這項技術本質上是好的，只是人性的貪婪造成了這些負面影響。

**Comment by wh20502** [comment]
可以去研發Deepfake的Deepfake，既然人沒辦法辨識，這樣就只能讓程式互鬥了

**Comment by ghost** [comment]
只能說  看到這種好像天上掉下前來的機會 都要特別的警惕

**Comment by jack90325** [comment]
只能說這個世界上壞人太多了，不管開發出什麼科技都可以運用在壞事上

**Comment by ayakasanma** [comment]
Deepfake也很常用在謎片上，甚至可以以外流之名行抹黑之實，真的十分可怕

**Comment by Yunhans** [comment]
之前還看過歐巴馬的影片，根本分不出真偽

**Comment by lx3211** [comment]
感覺以後都不能相信影片內的事物了

**Comment by xCk27x** [comment]
https://www.thenewslens.com/article/157849
目前也有許多團隊在研發如何反制deepfake
例如芝加哥大學電腦科學系實驗室開發的「AI軟體Fawkes」
能你的照片加上一些「雜訊」，讓電腦無法辨識你的真實面容

**Comment by limitx0** [comment]
希望 AI 產生的問題也能用 AI 解決啊

**Comment by boson13579** [comment]
近年有很多app是關於人臉轉換，主打能根據照片將性別轉換，或利用照片將人臉轉到短影片的人臉上，這種新穎的體驗方式讓許多年輕的消費者感興趣，卻也在不知不覺之中將自己的肖像流於網路上，讓非法份子有機可乘。因此，應提高警覺，盡量減少接觸會讓自己肖像外露的app。

**Comment by larry40305** [comment]
現在有很多利用Deepfake的犯罪，政府必須要立法加以規範。

**Comment by HsuChen88** [comment]
Deepfake的出現等於把認證這件事推翻了，以往我們使用的認證方式若都能被假冒的話，認證的目的都已成為泡沫。

**Comment by ghost** [comment]
相信Deepfake這技術本來一定是好的，但是由於人類的不當行為使的他污名化，這也警惕了我們技術並沒有好壞之分，取決於使用者如何應用他們。

**Comment by Yilin20021016** [comment]
但deepfake有太多人用在不合法的東西上了

**Comment by titan92601** [comment]
Deepfake 的價值大家都清楚，差別在於人們使用的方式不同，科學既然已經提供給我們如此良好的東西為何要用來傷害他人或抹黑其價值呢，只能說在動小聰明以前先好好想想吧！

**Comment by Vincent20030314** [comment]
deepfake是個方便的東西，但是被用在不合法的用途上很可惜，希望之後人們能改變用途。
